src/radix-trie.ts

```diff
@@ -22,24 +22,24 @@ export class Leaf<T> {
   }
 }
 
-export class RadixTrie {
```

Comment:
This change makes the code a lot more readable and being able to subsequently remove the `unknown` types is great 🎉